plugins/qtAccessionPlugin/lib/model/map/AccessionEventI18nTableMap.php

```diff
@@ -1,18 +1,22 @@
 <?php
 
-
-/**
- * This class defines the structure of the 'accession_event_i18n' table.
- *
+/*
+ * This file is part of the Access to Memory (AtoM) software.
  *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
  *
- * This map class is used by Propel to do runtime db structure discovery.
- * For example, the createSelectSql() method checks the type of a given column used in an
- * ORDER BY clause to know whether it needs to apply SQL to make the ORDER BY case-insensitive
- * (i.e. if it's a text column type).
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
  *
- * @package    plugins.qtAccessionPlugin.lib.model.map
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
  */
```

Comment:
This will probably be removed the next time we build the models. I think we don't add the default comment in this classes for that reason.